bitnami/etcd/templates/statefulset.yaml

```diff
@@ -94,52 +94,47 @@ spec:
             }
 
             ## Create data dir if not exists
-            if [ ! -d '${DATA_DIR}' ]; then
+            if [ ! -d ${DATA_DIR} ]; then
```

Comment:
Double quote to prevent globbing and word splitting: `if [ ! -d "${DATA_DIR}" ]; then`